habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml

```diff
@@ -0,0 +1,56 @@
+# @package _global_
+
+defaults:
+  - /habitat: habitat_config_base
+  - /habitat/simulator/agents@habitat.simulator.agents.main_agent: rgbdp_head_rgb_third_agent
+  - /habitat/task/rearrange: cat_gaze
+  - /habitat/dataset/rearrangement: floorplanner
+  - _self_
+
+habitat:
+  gym:
+    auto_name: Pick
+    obs_keys:
+      - robot_head_rgb
+      - robot_head_depth
+      - obj_start_sensor
+      - object_embedding
+      - joint
+      - is_holding
+      - relative_resting_position
+  environment:
+    max_episode_steps: 300
+  simulator:
+    type: ObjectRearrangeSim-v0
+    additional_object_paths:
+      - data/objects/ycb/configs/
+      - data/objects/amazon_berkeley/configs/
+      - data/objects/google_object_dataset/configs/
+    needs_markers: False
+    concur_render: True
+    auto_sleep: True
+    agents:
+      main_agent:
+        radius: 0.25
```

Comment:
Shouldn't this be 0.3?

Reply:
I guess this should be 0.085 now? Can we also add height here?

Reply:
Added height, 0.085 will come with Teleport action changes.